### ISSUE TITLE
fail to open bam file at baseline step

### ISSUE THREAD

**Issue by arielyh** [xjtu-omics/msisensor-pro#12]
I'm using a cluster and installed msisensor-pro via conda. Here is my command and results:
(msisensor-env) [myid@compute-a-xx-xxx msisensor_test]$ **msisensor-pro baseline -d hg19_msi_reference.list -i configure.txt -o .
baseline -d hg19_msi_reference.list -i configure.txt -o .**

Check for the environment ...
Current work path: /home/myid/msisensor_test
Microsatellites file: /home/myid/msisensor_test/hg19_msi_reference.list
Configure file path: /home/myid/msisensor_test/configure.txt
Output path:/home/myid/msisensor_test
/home/myid/msisensor_test/ is existing! OK!
/home/myid/msisensor_test/detail is existing! OK!

Load files ...
load bam:/home/myid/msisensor_test/normal/case1_sorted.bam OK!
load bam:/home/myid/msisensor_test/normal/case2_sorted.bam OK!
Open bam file failed( ), please provide valid bam file with its index file !

However, I have both the .bam and .bai files under the normal folder. I've also checked the bam files using samtools and they are correct with completed EOF. I tried both case1_sorted.bam.bai and case1_sorted.bai, but neither of them worked for me.

**Comment by PengJia6** [comment]
@arielyh  I am really sorry to hear that.  There may be a bug in bam file reading process of msisensor-pro. Could you please run msisenor-pro with the demo data we provided on [github](https://github.com/xjtu-omics/msisensor-pro/tree/master/demo). If it is still failed, please give me a test bam file so that I could repeat the error. 

Please concat with me if any question!

**Comment by arielyh** [comment]
@PengJia6 The demo data works fine for me. I tried both ./tumor_only.sh and ./tumor_normal.sh and they finished successfully. I don't think I can share the bam file. I sorted the bam file by coordinate, not by name. Is this correct?

**Comment by PengJia6** [comment]
@arielyh  This is correct to sort by coordinate.   

 I just update the program, you can try the new version (1.0.2). Please tell me if the new version solve you proplem.  

_Note: You can [use the binary file](https://github.com/xjtu-omics/msisensor-pro/wiki/How-to-install-MSIsensor-pro#directly-using-binary-version) directly or [make by your self](https://github.com/xjtu-omics/msisensor-pro/wiki/How-to-install-MSIsensor-pro#install-from-source-code) to install the new version now, and it will be supported by conda later._

**Comment by arielyh** [comment]
@PengJia6 
Unfortunately, I cannot install the binary file. When I followed the instruction of binary file, I got the following error:
msisensor-pro: /lib64/libstdc++.so.6: version `GLIBCXX_3.4.20' not found (required by msisensor-pro)
msisensor-pro: /lib64/libstdc++.so.6: version `CXXABI_1.3.9' not found (required by msisensor-pro)
msisensor-pro: /lib64/libstdc++.so.6: version `GLIBCXX_3.4.21' not found (required by msisensor-pro)

I've also tried to make by myself. However I don't have permission to "mv msisensor-pro /usr/local/bin/" on the cluster.

The only installation option worked for me is through conda install. I probably will wait until the conda update available and try it. 

BTW, I tried the original msisensor version on my normal-tumor pairs and it worked fine. So my bam files are good. Thank you very much.

**Comment by PengJia6** [comment]
@arielyh Sorry to hear that again! I will update the conda version within 24 hours. By the way ,for source code version,  if you make successfully, you can use the absolute path of msisensor-pro: 

       /path/to/msisensor-pro/cpp/msisensor-pro baseline -d hg19_msi_reference.list -i configure.txt -o .

**Comment by arielyh** [comment]
@PengJia6 
The 1.0.2 version works now!!! I tried the absolute path after I make by myself. The index bai files need to be named as case1_sorted.bam.bai, NOT case1_sorted.bai and then it worked for me. 
I also deleted an empty line at the end of my configure.txt file. I'm not sure whether it's this empty line issue or the case1_sorted.bai issue. But it worked for me anyway. Thank you soooo much for your quick response!
